src/app/app.module.ts

```diff
@@ -21,7 +24,9 @@ import {AddProjectComponent} from './admin/add-project/add-project.component'
     UserObservationsComponent,
     LoginComponent,
     UserDashboardComponent,
-    AddProjectComponent
+    MakeObservationComponent,
+    AddProjectComponent,
+    UpdateObservationComponent
```

Comment:
These should be declared in the shared module - It's best practice to not import individual components at the app level when they are housed in a feature module.